### ISSUE TITLE
Intermittent sound in Firefox

### ISSUE THREAD

**Issue by bflora2** [goldfire/howler.js#474]
I've had a user complain that the sound effects in my HTML5 game will sometimes stop playing in the middle of the game in Firefox (Mozilla/5.0 (Windows NT 6.1; rv:44.0) Gecko/20100101 Firefox/44.0;). My sound effect is a simple "click" sound using Howler 2.0 beta 8:

```
var sound = new Howl({
    src: ['sound.ogg', 'sound.mp3'],
    volume: 0.3
});

sound.play();
```

While investigating the issue with Firefox on my PC (Mozilla/5.0 (Windows NT 6.3; WOW64; rv:44.0) Gecko/20100101 Firefox/44.0;) I saw the same problem happen once. I traced execution down to line `node.bufferSource.start(0, seek, duration)` in Howler's Play function, the same execution path as when the sound plays properly. The sound started working again when I restarted Firefox. So, I think this is probably a Firefox issue, but I'm curious if this is a known issue? Is there a workaround, or anything I should try? Is there a format that works more reliably on Firefox than OGG and MP3?

**Comment by goldfire** [comment]
I don't know if it is related to this, but I now recommend using WebM instead of OGG as there are some known issues with OGG (especially in Firefox). I haven't heard of this happening before, but it is possible it is related.

https://github.com/goldfire/howler.js/tree/2.0#format-recommendations

**Comment by bflora2** [comment]
Thanks, James! I had more trouble finding a free tool to convert to webm than I expected, but I finally found one called Xmedia Recode. The resulting webm file works with Firefox, but Chrome can not decode it, so I'll have to use the file with Firefox only. What do you use as your preferred tool for webm files?

I'll report back on whether webm fixes my user's intermittent sound problem in case you find the information useful.

**Comment by goldfire** [comment]
WebM should work anywhere that OGG works. I use ffmpeg for everything audio related.

**Comment by bflora2** [comment]
FYI... My user got back to me and confirmed that switching to webm sound effects fixed the intermittent sound problem he was experiencing on Firefox. So, thank you for the webm tip, James!

I tried FFmpeg to create the webm sounds, but Chrome still could not decode them. I noticed both Xmedia Recode and FFmpeg default to the Opus codec for encoding webm, so I tried switching to the Vorbis codec instead. That fixed the problem and now the webm sounds work fine on Chrome.
